pkg/operator/sync.go

```diff
@@ -326,7 +320,7 @@ func (optr *Operator) syncRequiredMachineConfigPools(_ *renderConfig) error {
 	}
 
 	var lastErr error
-	if err := wait.Poll(time.Second, 10*time.Minute, func() (bool, error) {
+	if err := wait.Poll(time.Second, 15*time.Minute, func() (bool, error) {
```

Comment:
Use a const like the other timeouts? e.g. https://github.com/openshift/machine-config-operator/blob/eab95d598b9b32361d0ea971906e2ebe93217201/pkg/operator/sync.go#L375